ipam.go

```diff
@@ -73,7 +74,11 @@ func (c *ipamCache) match(prefix string) *ipPool {
 // update updates the internal map with IPAM updates when the update
 // is new addtion to the map or changes the existing item, it calls
 // updateHandler
-func (c *ipamCache) update(node *etcd.Node, del bool) error {
+func (c *ipamCache) update(nodeEtcd interface{}, del bool) error {
```

Comment:
This function is a ipamCache internal function and isn't called from outside.
Why do you need this change?

Reply:
update is the implement of IpamCache interface type.